index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kamulete training center</title>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">    
+    <script src="https://kit.fontawesome.com/ecb44d8fff.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <nav class="twitter">
+            <a href="#"><i class="fa-brands fa-square-twitter" width="10px"></i></a>
+            <a href="#">Whatsap</a>
+            <a href="#">training</a>
+            <a href="#">My page</a>
+            <a href="#">logout</a>
+        </nav>
+        <nav class="navlogo">
+            <a class="img-logo" href="index.html"><img src="img-folder/logo-image---00.png" alt=""></a>
+            <div class="hamburg">
+                <div class="hamburg1"></div>
+                <div class="hamburg2"></div>
+                <div class="hamburg3"></div>
+            </div>
+            <ul class="menu">
+                <li><a class="menu-item" href="index.html">home</a></li>
+                <li><a class="menu-item" href="#programs">program</a></li>
+                <li><a class="menu-item" href="#ourTeacher">Teachers</a></li>
+                <li><a class="menu-item" href="About.html">About</a></li>
+            </ul>    
+        </nav>
+    </header>
+        <section class="kamuclass">
+            <p class="hello">" Hello! Are you intersted <br>to Speak many languages?"</p>
```

Comment:
- [ ] As mentioned by the previous reviewer please make sure you `<p> Hello! Are you intersted <br>to Speak many languages?"</p>` doesn't have a break and fits in one line to match design guidelines

Your design:

<img width="301" alt="Screenshot 2023-03-16 at 9 50 19 PM" src="https://user-images.githubusercontent.com/44798044/225729918-e16f3810-1826-4d83-ad91-9e8cb3067633.png">

Design guidelines:
<img width="329" alt="Screenshot 2023-03-17 at 12 18 20 AM" src="https://user-images.githubusercontent.com/44798044/225730048-b019ec76-d46d-40f7-8c21-746286445ffa.png">